src/ttsnake.c

```diff
@@ -498,6 +498,7 @@ void * userinput(){
 				break;
       			case KEY_UP:
       			case 'w':
+				case 'k':
 			case 'e':
```

Comment:
We have one extra tab here:
```suggestion
      			case 'w':
			case 'k':
			case 'e':
```